apps/web/src/index.css

```
@@ -348,3 +348,89 @@ body {
+    padding: 0 !important;
+    min-height: 0 !important;
+    height: auto !important;
+    overflow: visible !important;
+  }
+
+  [data-print-stack] {
+    display: block !important;
+  }
+
+  [data-print-root] {
+    page-break-after: always;
```

Stylelint (17.14.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)